weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/DataTableView.tsx

```diff
@@ -0,0 +1,340 @@
+import LinkIcon from '@mui/icons-material/Link';
```

Comment:
Most of this file is extracting what was still useful from the now deleted `weave-js/src/components/PagePanelComponents/Home/Browse2/WeaveEditors.tsx`